prime-angular-frontend/src/app/core/resources/site-resource.service.ts

```diff
@@ -55,9 +56,14 @@ export class SiteResource {
 
   public getSiteById(siteId: number, statusCode?: number): Observable<Site> {
     const params = this.apiResourceUtilsService.makeHttpParams({ statusCode });
-    return this.apiResource.get<Site>(`sites/${siteId}`, params)
-      .pipe(
-        map((response: ApiHttpResponse<Site>) => response.result),
+    return forkJoin({
+      site: this.apiResource.get<Site>(`sites/${siteId}`, params)
+        .pipe(map((response: ApiHttpResponse<Site>) => response.result)),
```

Comment:
im assuming the next line i can do the same thing?
``` this.apiResource.get<IndividualDeviceProvider[]>(`sites/${siteId}/individual-device-providers`, null, null, true) ```

Reply:
Yeah definitely, and in PIP it would be `this.apiResource.get<IndividualDeviceProvider[]>('sites/${siteId}/individual-device-providers')` so we won't need the extra params in the near future